src/components/SecuritySection.vue

```diff
@@ -0,0 +1,203 @@
+<!--
+  - SPDX-FileCopyrightText: 2022 Carl Schwan <carl@carlschwan.eu>
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+  -->
+
+<template>
+	<SettingsSection :title="t('end_to_end_encryption', 'End to End Encryption')"
```

Comment:
```suggestion
	<SettingsSection :title="t('end_to_end_encryption', 'End to end encryption')"
```
"Sentence case" instead of "capital case"